app/src/androidTest/java/com/github/se/signify/ui/screens/profile/ProfileScreenTest.kt

```diff
@@ -129,4 +125,35 @@ class ProfileScreenTest {
     composeTestRule.onNodeWithTag("HelpButton").performClick()
     composeTestRule.onNodeWithTag("HelpPopupContent").assertIsDisplayed()
   }
+
+  @Test
+  fun statsAreDisplayed() {
+    // Verify exercises achieved section is displayed
+    composeTestRule.onNodeWithTag("ExercisesColumn").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("ExercisesRow").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("ExercisesText").performScrollTo().assertIsDisplayed()
+    composeTestRule
+        .onNodeWithTag("ExercisesText")
+        .performScrollTo()
+        .assertTextEquals("Number of exercises achieved :")
+    composeTestRule.onNodeWithTag("Easy").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Easy").performScrollTo().assertTextEquals("Easy")
+    composeTestRule.onNodeWithTag("Medium").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Medium").performScrollTo().assertTextEquals("Medium")
+    composeTestRule.onNodeWithTag("Hard").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Hard").performScrollTo().assertTextEquals("Hard")
+
+    // Verify quests achieved section is displayed
+    composeTestRule.onNodeWithTag("QuestsColumn").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("QuestsRow").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("QuestsText").performScrollTo().assertIsDisplayed()
+    composeTestRule
+        .onNodeWithTag("QuestsText")
+        .performScrollTo()
+        .assertTextEquals("Number of quests achieved :")
+    composeTestRule.onNodeWithTag("Daily").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Daily").performScrollTo().assertTextEquals("Daily")
+    composeTestRule.onNodeWithTag("Weekly").performScrollTo().assertIsDisplayed()
+    composeTestRule.onNodeWithTag("Weekly").performScrollTo().assertTextEquals("Weekly")
```

Comment:
Tests should never be checking for exact text.
You've merely moved this test, so it's not your fault. We have hardcoded text assertions everywhere already anyway.
Ideally, these tests would use string resources.